cnd/src/swap_protocols.rs

```diff
@@ -1,10 +1,11 @@
 pub mod actions;
 mod facade;
 pub mod halight;
-pub mod han;
 pub mod herc20;
+pub mod hneth;
 pub mod ledger;
 pub mod ledger_states;
+pub mod lnbtc;
```

Comment:
I like the new naming, good step forward :)

I am sort of missing the `H` in `LNBTC` though. To signal compatibility with other HTLC based swaps, I think this one should say it as-well.
Also, if LN ever gets scriptless script support, then we will need a different protocol but it would still be Bitcoin on Lightning, just a different version of lnd probably.

Reply:
`hlnbtc`?

Reply:
Changed to use `hlnbtc`.  These 'words' are a little hard to parse in code (.e.g., `HnethHlnbtcFoo`), not sure if our eyes will get used to parsing them of if we want to mess with the format a bit e.g., `HnEth` but I'm hesitant to do this because it violates idiomatic rust (how to capitalise acronyms).  Any thoughts?

Reply:
I'd suggest `haneth` and not `hneth`. The removal of one vowel just makes it worse.

Reply:
> I'd suggest haneth and not hneth. The removal of one vowel just makes it worse.

Not a bad suggestion, this makes `hlnbtc` and `hanbtc` have the same number of characters, which is nicer.  Also makes the possibility of writing identifiers as `HanBtc` more palatable.  Are there any arguments _against_ `haneth`/`hanbtc`?